src/Model.php

```diff
@@ -1676,9 +1676,11 @@ public function insert(array $row)
      */
     public function import(array $rows)
     {
-        foreach ($rows as $row) {
-            $this->insert($row);
-        }
+        $this->atomic(function () use ($rows) {
```

Comment:
now import is now only safer, but much faster ;-)

Reply:
I wonder why didn't we fix this ages ago. Multiple inserts in single transaction should give noticeable performance improvement.

Reply:
they were not in single transaction, but for each import a new transaction was started :D

Reply:
as major, moved to saparate PR - https://github.com/atk4/data/pull/767